deploy/chart/kuiper/templates/deployment.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ekuiper.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "ekuiper.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "ekuiper.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "ekuiper.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "ekuiper.serviceAccountName" . }}
+      volumes:
+      - name: ekuiper-data
+        {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: {{ (tpl .Values.persistence.existingClaim $) | default (include "ekuiper.fullname" .) }}
+        {{- else }}
+        emptyDir: {}
+        {{- end }}
+      - name: kuiper-config
+        configMap:
+          name: {{ include "ekuiper.fullname" . }}
+          items:
+          - key: kuiper.yaml
+            path: kuiper.yaml
+      {{- if .Values.tls.enabled }}
+      - name: certs
+        secret:
+          secretName: {{ template "neuron.tlsSecretName" . }}
+          items:
+            - key: {{ ternary "tls.crt" "ca.crt" .Values.tls.existingSecretFullChain }}
+              path: xyz-rootca.pem
+            - key: tls.crt
+              path: xyz-certificate.pem
+            - key: tls.key
+              path: xyz-private.pem.key
+        {{- end }}
+      containers:
+        - name: kuiper
+          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}-slim"
```

Comment:
```suggestion
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}-slim"
```

Reply:
I'm not sure we need to hardcode the slim image ?

Reply:
Jiyong suggested that we'd better use slim.